classes/People.php

```diff
@@ -113,6 +113,7 @@ protected function getCSVRow($details) {
     }
 
     public function sendAsCSV($data) {
+        header('Content-Disposition: attachment; filename=' . $this->rep_plural . '.csv');
```

Comment:
The rep_plural for lords is "Members of the House of Lords" which won't work without quotes, and might not be sensible to use anyway just in case. `$this->type` probably okay?